woocommerce/woocommerce
# Update Emogrifier to 2.0

This PR updates the Emogrifier class to 2.0.

Emogrifier requires DomDocument and PHP 5.5+. Checks have been added to include this only when these prerequisites are met.

If they are not met, this will load styles inline. 3.4 and below did not do this and just left emails unstyled.

Closes #20545

### How to test the changes in this Pull Request:

1. PHP 5.5+ trigger customer invoice email. Check styles are correct.
2. PHP 5.4. Repeat. Check style block is present inside email content.
3. Test case #20545 and ensure inline styles still render correctly.

### Changelog entry

> Updated Emogrifier to 2.0.
